src/components/EditableCategoryTitle/tests/EditableCategoryTitle.test.tsx

```diff
@@ -41,4 +47,25 @@ describe(`EditableCategoryTitle`, () => {
       })
     ).toBeInTheDocument();
   });
+
+  it(`Should render edit category dialog when clicking in edit category trigger button`, async () => {
+    vi.mock(`axios`);
+    const mockedAxios = axios as jest.Mocked<typeof axios>;
+    mockedAxios.get.mockResolvedValueOnce({
+      data: { products: [], gettingProducts: false },
+    });
+
+    renderWithRedux(<EditableCategoryTitle {...mockProps} />);
+
+    expect(screen.queryByRole(`dialog`)).not.toBeInTheDocument();
```

Comment:
```suggestion
    const dialog = screen.queryByRole(`dialog`);
    
    expect(dialog).not.toBeInTheDocument();
```
 screen.queryByRole(`dialog`) was used 2 times, you dont think is better use variable?

Reply:
This wont work, because the first time and the second are made in different situations. The test would fail.

Reply:
aaaah, nice